test_data/two-fer/imperfect_solution/expected_analysis.json

```diff
@@ -1 +1 @@
+{"summary":"Check the comments for some suggestions. 📣","comments":[{"comment":"elm.common.no_unused.custom_type_constructor_args","type":"actionable","params":{"definition":" 9| -- unused type constructor argument String\n10| type TwoFer = TwoFerUnused String\n                               ^^^^^^"}},{"comment":"elm.common.no_unused.custom_type_constructors","type":"actionable","params":{"definition":" 9| -- unused type constructor argument String\n10| type TwoFer = TwoFerUnused String\n                  ^^^^^^^^^^^^"}},{"comment":"elm.common.no_unused.parameters","type":"actionable","params":{"definition":"30| -- unused is unused\n31| unusedParameter unused =\n                    ^^^^^^\n32|   case Nothing of"}},{"comment":"elm.common.no_unused.patterns","type":"actionable","params":{"definition":"34|     -- something is unused\n35|     Just something -> 1 * 1\n             ^^^^^^^^^\n36|     -- True && False can be simplified to False"}},{"comment":"elm.common.no_unused.variables","type":"actionable","params":{"definition":"3| -- unused imported function\n4| import Maybe exposing (withDefault)\n                          ^^^^^^^^^^^\n5| -- unused imported module"}},{"comment":"elm.common.no_unused.variables","type":"actionable","params":{"definition":"5| -- unused imported module\n6| import Parser.Advanced\n          ^^^^^^^^^^^^^^^"}},{"comment":"elm.common.simplify","type":"actionable","params":{"message":"Simplify: Unnecessary multiplication by 1\n\n34|     -- something is unused\n35|     Just something -> 1 * 1\n                           ^^^^\n36|     -- True && False can be simplified to False\n\nMultiplying by 1 does not change the value of the number."}},{"comment":"elm.common.simplify","type":"actionable","params":{"message":"Simplify: Part of the expression is unnecessary\n\n37|     -- if condition can be simplified to 0\n38|     Nothing -> if (True && False) then 1 else 0\n                       ^^^^^^^^^^^^^\n\nA part of this condition is unnecessary. You can remove it and it would not impact the behavior of the program."}},{"comment":"elm.common.simplify","type":"actionable","params":{"message":"Simplify: Comparison is always False\n\n37|     -- if condition can be simplified to 0\n38|     Nothing -> if (True && False) then 1 else 0\n                       ^^^^^^^^^^^^^\n\nThis condition will always result in the same value. You may have hardcoded a value or mistyped a condition."}}]}
```

Comment:
Is this due to the update in elm-review?

Reply:
Yes! 
Some messages changed the wording a bit, and the order had changed for some reason (but I can't be mad, it's not like the order is guaranteed by the elm-review API) so I added a secondary order to our comments to help with that. 
I suppose approach would be to use `jq` to order the fields before comparing them but that might be a bit tricky too.